docs/src/physics/boussinesq.md

```diff
@@ -1,6 +1,6 @@
 # The Boussinesq approximation
 
-Oceananigans.jl employ often the Boussinesq approximation[^1]. In the Boussinesq approximation
+Oceananigans.jl often employs the Boussinesq approximation[^1]. In the Boussinesq approximation
```

Comment:
Oceananigans always makes the Boussinesq approxmation, unless I'm missing something?